plugins/mapper-annotated-text/src/yamlRestTest/resources/rest-api-spec/test/mapper_annotatedtext/20_synthetic_source.yml

```diff
@@ -0,0 +1,164 @@
+---
+setup:
+  - skip:
```

Comment:
I tried to use features but i could not figure it out. First of all it did not seem worth to add a feature flag for this and we are also inside the plugin and i don't know how to set up features properly.